ts/sagas/contentLoaders.ts

```diff
@@ -43,7 +43,7 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
       serviceId
     );
 
-    if (response.isRight() && response.value.status === 200) {
+    if (response && response.isRight() && response.value.status === 200) {
```

Comment:
same here, `getServiceMetadata` should always return an `Either` and should resolve to a `Left` at row 25